### PR TITLE
[Keyboard] Fix compile issues for OLKB Default keymaps

### PR DESCRIPTION
**Pull request by drashna**
## Description

This should fix the compile issues that I introduced by not properly testing out the boards. 

This time, they've been more thoroughly tested to ensure that they don't break anything. 

## Types of Changes

- [x] Bugfix
- [x] Keymap/layout/userspace (addition or update)

## Issues Fixed or Closed by This PR

* Travis, and flagged issue.

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/preonic/keymaps/default/keymap.c [hunk 1]
```diff
@@ -274,7 +274,7 @@ void dip_switch_update_user(uint8_t index, bool active) {
             } else {
                 muse_mode = false;
             }
-    xs}
+    }
```
Comment (fauxpark) [inline keyboards/preonic/keymaps/default/keymap.c]: This was fixed in https://github.com/qmk/qmk_firmware/pull/6748... but no merge conflict?

**Comment by fauxpark** [comment]
Travis tick can't be trusted here, it's failing to run the `:all` target but reporting success anyway, as the return code is 0.

File: keyboards/preonic/keymaps/default/keymap.c (hunk 1 above)
Comment (drashna) [reply keyboards/preonic/keymaps/default/keymap.c]: lol. Probably because it's the same change?

**Comment by drashna** [comment]
That's okay, this time I compiled EVERY version of the planck and preonic, just to make sure.

**Comment by fauxpark** [comment]
👍

**Pull request merged**